src/shared/helpers/urlHelper.ts

```diff
@@ -36,12 +35,53 @@ type SpeciesPageUrlParams = {
   genomeId: string;
 };
 
+type SpeciesSelectorParams = {
+  query?: string;
+  gene?: {
+    query?: string;
+  };
+  speciesTaxonomyId?: string | number;
+};
+
 export const speciesPage = (params: SpeciesPageUrlParams) => {
   const speciesPageRootPath = '/species';
 
   return `${speciesPageRootPath}/${params.genomeId}`;
 };
 
+export const speciesSelector = (params?: SpeciesSelectorParams) => {
```

Comment:
This should probably be split into three:
- `speciesSelector`: just returns `/species-selector`
- `speciesSelectorSearch`: returns `/species-selector/search?query...` or `/species-selector/search?species_taxonomy_id: ...`
- `geneSearch` returns whatever url we decided to use for gene search.